src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,31 @@
+import { memo } from 'react';
+import { useParams } from 'react-router-dom';
+import { tabs } from '../../api/tabs';
```

Comment:
Add this to the PR
![image](https://github.com/mate-academy/react_tabs-with-router/assets/91411692/c3801bd7-dc85-44ac-a450-66e35effd9a0)